scripts/EGAP/files_to_import_structure.py

```diff
@@ -0,0 +1,43 @@
+import os
+import shutil
+from distutils.dir_util import copy_tree
+BASE_PATH = 'EGAP_REGISTRY_staging/3 Registrations/'
```

Comment:
Similarly to my script, could you add args for specifying source/target for the files? The target path won't always be `EGAP_data_10-22-2019`